conda_rpms/tests/unit/generate/test_render_env.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+
+from conda_rpms.generate import render_env
+
+
+class Test_tag(unittest.TestCase):
+    def check(self, tag, expected):
+        config = {'install': {'prefix': '/data/local'},
+                  'rpm': {'prefix': 'Tools'}
+                  }
+        r = render_env(branch_name='default', label='next', config=config,
+                       tag=tag, commit_num=30)
+        result_requires = [
+            line for line in r.split('\n') if line.startswith('Requires: ')]
+        self.assertEqual(result_requires, expected)
+
+    def test_tag(self):
+        self.check(tag='env-default-2016_12_15',
+                   expected=['Requires: Tools-env-default-tag-2016_12_15'])
```

Comment:
@lbdreyer Perhaps add a simple test for a tag with an alphanumeric environment name i.e. `env-default1234-2016_12_15`